lib/ghom.gd

```diff
@@ -674,4 +674,4 @@ DeclareAttribute( "ImagesSmallestGenerators",
 ##
 DeclareAttribute( "RegularActionHomomorphism", IsGroup );
 
-DeclareGlobalName("IsomorphismAbelianGroupViaIndependentGenerators");
+DeclareGlobalFunction("IsomorphismAbelianGroupViaIndependentGenerators");
```

Comment:
The commit message says:

> Use DeclageGlobalFunction instead of DeclareName
>
> This is relevant when forming factor groups as subdirect products.

Could you elaborate? The only scenario I could think of is one where somewhere there is code which tries to store `IsomorphismAbelianGroupViaIndependentGenerators` somewhere before using it, i.e., does something like `x := IsomorphismAbelianGroupViaIndependentGenerators` or `x[1] := IsomorphismAbelianGroupViaIndependentGenerators;` or some such, but no such code seems to exist. So I am now wondering what I am missing

Reply:
Will edit. This is from the context in which the redeclaring came up, but is not really relevant anymore

Reply:
OK, the commit message now states:

> Use DeclareGlobalFunction instead of DeclareName, since that is the proper way

I disagree with the characterization of "this is the proper way". But I don't mind it being changed, either; as stated earlier, I wanted to understand why it might be necessary or "relevant" as the commit message stated before. If it's just for taste, and there is no underlying bug, I am content :-)